modules/ROOT/pages/view-health.adoc

```diff
@@ -15,4 +15,4 @@ For Runtime Fabric version 1.5 or later:
 The *Health Details* section indicates whether your Runtime Fabric cluster configuration meets the requirements for a production environment, a development environment, or neither. If your configuration fails to meet the minimum requirements of both production and development environments, `Not supported` is displayed. 
 
 == See Also
-You can view additional performance and Runtime Fabric cluster health information using the Ops Center. Refer to xref:using-opscenter[Use Ops Center to Monitor Runtime Fabric].
+You can view additional performance and Runtime Fabric cluster health information using the Ops Center. Refer to xref:using-opscenter.adoc[Use Ops Center to Monitor Runtime Fabric].
```

Comment:
Should the bracketed text be "use the Ops Center to Monitor Runtime Fabric"?

Reply:
I think so....